<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>972</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Perfm</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <widget class="QTreeView" name="treeView">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>9</y>
         <width>531</width>
         <height>421</height>
        </rect>
       </property>
      </widget>
      <widget class="QWidget" name="file_info_widget" native="true">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>20</y>
         <width>305</width>
         <height>234</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="file_name_label">
            <property name="text">
             <string>Filename:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="file_name_info">
            <property name="text">
             <string>Some_file.txt</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="file_size_label">
            <property name="text">
             <string>Filesize:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="file_size_info">
            <property name="text">
             <string>0 bytes</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="file_changed_label">
            <property name="text">
             <string>Last Changed:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="file_changed_info">
            <property name="text">
             <string>Sat 13 Feb 2021 03∶56∶05 PM EST</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="permissions_label">
          <property name="text">
           <string>Permissions:</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="permissions_label_user">
            <property name="text">
             <string>User:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="permissions_info_user">
            <property name="text">
             <string>'r', 'w', '-'</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="permissions_label_group">
            <property name="text">
             <string>Group:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="permissions_info_group">
            <property name="text">
             <string>'r', 'w', '-'</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="permissions_label_other">
            <property name="text">
             <string>Other:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="permissions_info_other">
            <property name="text">
             <string>'r', 'w', '-'</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="octal">
           <attribute name="title">
            <string>Octal</string>
           </attribute>
          </widget>
          <widget class="QWidget" name="permission">
           <attribute name="title">
            <string>bitwise</string>
           </attribute>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>972</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
